livre/core/types.h

```diff
@@ -237,6 +237,7 @@ const uint32_t NODEID_FRAME_BITS = 18; //>! @see NodeId
 const uint32_t INVALID_POSITION = ( 1u << NODEID_BLOCK_BITS ) - 1; //!< Invalid node ID.
 const uint32_t INVALID_LEVEL = ( 1u << NODEID_LEVEL_BITS ) - 1; //!< Invalid tree level.4 bits is on
 const uint32_t INVALID_FRAME = ( 1u << NODEID_FRAME_BITS ) - 1; //!< Invalid tree level.4 bits is on
+const uint32_t LATEST_FRAME = INT_MAX;
```

Comment:
http://www.cplusplus.com/reference/climits/

If we are getting INT_MAX from here, the limits are as given in the header and it is ushort. If we are getting from somewhere else, be sure that it is INT_MAX. I did accept as si before but the definitions of it is mixed. If we defined it, it is even worse.